docs/how-to/document_search/search_documents.md

```diff
@@ -14,7 +15,8 @@ from ragbits.core.embeddings.litellm import LiteLLMEmbeddings
 from ragbits.core.vector_stores.in_memory import InMemoryVectorStore
 from ragbits.document_search import DocumentSearch
 from ragbits.document_search.documents.document import DocumentMeta
-from ragbits.document_search.documents.sources import GCSSource
+from ragbits.document_search.documents.sources.gcs import GCSSource
```

Comment:
Cann't we just export the various sources from the main `ragbits.document_search.documents.sources` package? (i.e., the `__init__.py` file there)